storybook/src/components/Dialog/Dialog.docs.mdx

```diff
@@ -24,14 +24,14 @@ Click or tap the button to open a dialog.
 
 <Canvas of={DialogStories.TriggerButton} />
 
-Some basic example code to open and close a dialog:
+You can open the dialog using this code:
 
 ```ts
 const openDialog = () => (document.querySelector("#openDialog") as HTMLDialogElement)?.showModal();
```

Comment:
How about complementing our function to close a dialog with one to open it, e.g. `openDialog('#dialogId')`?